packages/web-react/src/components/Breadcrumbs/stories/BreadcrumbsItem.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+import { Breadcrumbs, BreadcrumbsItem } from '..';
+
+const meta: Meta<typeof BreadcrumbsItem> = {
+  title: 'Components/Breadcrumbs',
+  component: BreadcrumbsItem,
+  argTypes: {
+    iconNameStart: {
+      control: 'text',
+    },
+    iconNameEnd: {
+      control: 'text',
```

Comment:
Can add defaults here and to Start icon as well? And sort them 🔡

Reply:
Fixed.